### ISSUE TITLE
Depths > 10,000 ft Rounded On Write

### ISSUE THREAD

**Issue by toddheitmann** [kinverarity1/lasio#232]
# Issue

When writing logs greater than 10,000 ft to an las file, the depths are rounded to whole feet values rather than half feet. Below is example code and output. Data is attached as a txt file because GitHub does not allow .las file types to be uploaded.

## code

```python
import lasio

### orginal las file ###
log = lasio.read('TestLas.las')
print('Original Depths')
print(log[0])

### save las file and re-read ###
with open('TestOutput.las', 'w') as f:
    log.write(f, version = 2.0)

output_log = lasio.read('TestOutput.las')

print('New Depths After Writing')
print(output_log[0])
```

**Output**
```
Original Depths
[ 9995.5  9996.   9996.5  9997.   9997.5  9998.   9998.5  9999.   9999.5
 10000.  10000.5 10001.  10001.5 10002.  10002.5 10003.  10003.5 10004.
 10004.5 10005.  10005.5]
New Depths After Writing
[ 9995.5  9996.   9996.5  9997.   9997.5  9998.   9998.5  9999.   9999.5
 10000.  10000.  10001.  10002.  10002.  10002.  10003.  10004.  10004.
 10004.  10005.  10006. ]
```

Notice the printed values greater than 10,000 ft are rounded to integers and not half foot intervals.

Does anyone have an idea where to start looking for the fix?

## Data
[Click Here](https://github.com/kinverarity1/lasio/files/1929680/TestLas.txt) to download the data as a txt file. Las files cannot be uploaded to the issue, so I made the extension a txt file.

**Comment by kinverarity1** [comment]
This was fixed in #228 but I forgot to make a pypi release. Will do that shortly.

In the meantime you can specify the format with ``las.write(..., fmt="%.5f")``

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by toddheitmann** [comment]
Great! Thanks!
